b2sdk/transfer/inbound/downloaded_file.py

```diff
@@ -50,12 +53,13 @@ class MtimeUpdatedFile(io.IOBase):
        #  'some_local_path' has the mod_time set according to metadata in B2
     """
 
-    def __init__(self, path_, mod_time_millis: int, mode='wb+', buffering=None):
+    def __init__(self, path_, mod_time_millis: int, preferred_mode='wb+', buffering=None):
```

Comment:
please keep the old name

I don't see a good reason to introduce a API breaking change here. Just make default `None` for ppl that did not care about the exact mode set.